src/lib.rs

```diff
@@ -41,8 +41,12 @@
 //!
 //! ```
 //! # use xshell::cmd;
-//! let output = cmd!("date --iso").read()?;
-//! assert!(output.chars().all(|c| "01234567890-".contains(c)));
+//! if cfg!(target_os = "macos") {
+//!     let output = cmd!("date").read()?;
+//! } else {
+//!     let output = cmd!("date --iso").read()?;
+//!     assert!(output.chars().all(|c| "01234567890-".contains(c)));
+//! }
```

Comment:
Let's revert this change for now.

I haven't put time into fixing the tests for other OSes yet, but the way to do this would be to implement dummy `date` and `echo` ourselves, as binaries in `example`.